.gitignore

```
@@ -0,0 +1 @@
+package.xpkg
```

nitpick: likely don't ever want to commit an xpkg
```suggestion
*.xpkg
```